### PR TITLE
Saboteur borg module (syndicate engiborg)

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
WIP as fuck disclaimer here

tldr; it has no weapons and i've thrown out some of the modules it won't need, but it has thermals, a little bit better RCD and the unique ability to quicktravel via the disposals system as if it was a parcel.

:cl: WJohn & Naksu
add: Nuke ops may now order a borg with the Syndicate Saboteur module. It's a streamlined engineering unit capable of extraordinary robustness by virtue of not only not being able to leave its welder in the shuttle, but also quick travel around the station via the disposal system. Oh and it can also disguise itself.
rscadd: Syndicate medical and assault modules have received improved sprites
/:cl:

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
Related : https://cdn.discordapp.com/attachments/484170915253321734/487771927235002369/wave2.mp4

**Comment by TerraGS** [comment]
Since borgs being able to weld themselves is fine this should have no trouble getting merged.

**Comment by Therapist1** [comment]
Is that inbuilt reality distortion I see? (no but what’s the deal with that?)

**Comment by vuonojenmustaturska** [comment]
image
CUTE AS FUCK

**Comment by subject217** [comment]
by the way from the front and rear views it looks like the borg has cat ears, i'll presume this is intentional

**Comment by vuonojenmustaturska** [comment]
Not cat ears, they're the things on the side. 

Also, https://i.imgur.com/2MRLK5I.mp4

**Comment by zxaber** [comment]
That disguise effect is pretty neat.
What name does the borg take when disguised? Random station borg name, the player's normal borg name, or is it still syndicate-cyborg123?

**Comment by davethwave** [comment]
clicking the chameleon button too many times crashes the server.

**Comment by zombiegums** [comment]
Looks cool as hell. Love the disguise animation and pretty much everything about this.

**Comment by lolman360** [comment]
give it ventcrawl as well

**Comment by GunHog** [comment]
Why does this borg not get a weapon?

**Comment by lordsaladin** [comment]
will this be in the polymorph pool?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/silicon_defense.dm [hunk 1]
```diff
@@ -65,6 +65,9 @@
 
 //ATTACK HAND IGNORING PARENT RETURN VALUE
 /mob/living/silicon/attack_hand(mob/living/carbon/human/M)
+	. = FALSE
+	if(SEND_SIGNAL(src, COMSIG_ATOM_ATTACK_HAND, M) & COMPONENT_NO_ATTACK_HAND)
+		. = TRUE
```
[the remark this answers is not in the document]
Comment (vuonojenmustaturska) [reply code/modules/mob/living/silicon/silicon_defense.dm]: the parent is on /atom and does machine interaction

[the remark this answers is not in the document]
Comment (vuonojenmustaturska) [reply code/modules/mob/living/silicon/silicon_defense.dm]: the parent does non-silicon machine interaction
